scripts/environment/Dockerfile

```diff
@@ -10,7 +10,8 @@ ENV DEBIAN_FRONTEND=noninteractive \
 RUN echo $TZ > /etc/timezone
 
 # Setup the env
-RUN mkdir -p /git/timberio/vector/{scripts,scripts/environment}
+RUN mkdir -p /git/timberio/vector/scripts
```

Comment:
I think you can just delete this line no?

Reply:
There's a bug in there and the {} don't seem to work right.

Reply:
The `-p` flag on the next line creates the paths recursively as needed, so creating  `/git/timberio/vector/scripts` separately is not required